### PR TITLE
Feat/edit and remove coupon

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

**Comment by femacedogomes** [comment]
To test the update and remove  coupon component u need to be logged in supplier activated account and create your own coupon.
u can use  
username: sup_felipe@email.com
password: qweqwe123
page: https://deploy-preview-47--ez2live.netlify.app/dashboard/supplier/profile

File: src/components/mols/CouponActived/index.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import React from "react";
 import Image, { StaticImageData } from "next/image";
 import { AccordionInfo } from "@/components/atoms";
-import useDateFormater from "@/hooks/useDateFormater";
+import useDateFormater from "@/utils/DateFormater";
```
Comment (pedrohbl03) [inline src/components/mols/CouponActived/index.tsx:6]:
```suggestion
import getDeteFormater from "@/utils/DateFormater";
```

File: src/utils/DateFormater.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import dayjs from "dayjs";
 
-const useDateFormater = (date: string) => {
+const DateFormater = (date?: string) => {
```
Comment (pedrohbl03) [inline src/utils/DateFormater.ts:3]:
```suggestion
export const getDateFormater= (date?: string) => {
```

Comment (pedrohbl03) [reply src/utils/DateFormater.ts:3]: Change the file name to "getDateFormater" too

File: src/utils/DateFormater.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   const dataFormated = dayjs(date).format("DD/MM/YYYY");
   return dataFormated;
 };
-export default useDateFormater;
+export default DateFormater;
```
Comment (pedrohbl03) [inline src/utils/DateFormater.ts:7]:
```suggestion

```
remove this

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]

**Review by pedrohbl03 — APPROVED** [review]

**Comment by pedrohbl03** [comment]
@renanpontez @femacedogomes  closed

**Pull request merged**